test/jdk/java/foreign/TestLayouts.java

```diff
@@ -266,11 +266,13 @@ public void testPaddingIllegalByteSize() {
 
     @Test
     public void testStructToString() {
-        StructLayout padding = MemoryLayout.structLayout(JAVA_INT).withName("struct");
-        String basicLayoutString = (JAVA_INT.order() == ByteOrder.LITTLE_ENDIAN) ? "[i4](struct)" : "[I4](struct)";
-        assertEquals(padding.toString(), basicLayoutString);
-        var toStringUnaligned = padding.withByteAlignment(8).toString();
-        assertEquals(toStringUnaligned, "8%" + basicLayoutString);
+        for (ByteOrder order : List.of(ByteOrder.LITTLE_ENDIAN, ByteOrder.BIG_ENDIAN)) {
```

Comment:
The IBM copyright notice should be removed from this file as there are no other changes compared with upstream.

Reply:
Removed